src/main/scala/it/pagopa/interop/purposeprocess/api/impl/ResponseHandlers.scala

```diff
@@ -12,6 +12,17 @@ object ResponseHandlers extends AkkaResponses {
 
   implicit val serviceCode: ServiceCode = ServiceCode("012")
 
+  def retrieveLatestRiskAnalysisConfigurationResponse[T](logMessage: String)(
+    success: T => Route
+  )(result: Try[T])(implicit contexts: Seq[(String, String)], logger: LoggerTakingImplicit[ContextFieldsToLog]): Route =
+    result match {
+      case Success(s)                                           => success(s)
+      case Failure(ex: TenantNotFound)                          => notFound(ex, logMessage)
```

Comment:
If we do not find the tenant that is making the request, a 500 is the most appropriate response
```suggestion
```